src/injections/js/bug1605611-maps.google.com-directions-time.js

```diff
@@ -5,22 +5,15 @@
 /**
  * Bug 1605611 - Cannot change Departure/arrival dates in Google Maps on Android
  *
- * This patch does the following:
- * 1. Re-enable the disabled "Leave now" button.
- * 2. Fix the precision of datetime-local inputs (to minutes).
- * 3. Fixup side effect from enabling the date picker UI via
- *    injections/css/bug1605611-maps.google.com-directions-time.css
+ * This patch re-enables the disabled "Leave now" button.
  *
- * See https://bugzilla.mozilla.org/show_bug.cgi?id=1605611#c0 for details.
+ * See https://bugzilla.mozilla.org/show_bug.cgi?id=1605611 for details.
  */
 
-// Step 1.
 document.addEventListener("DOMContentLoaded", () => {
   // In case the element appeared before the MutationObserver was activated.
-  for (const elem of document.querySelectorAll(
-    ".ml-directions-time[disabled]"
-  )) {
-    elem.disabled = false;
+  for (const elem of document.querySelectorAll(".ml-icon-access-time")) {
```

Comment:
This intervention stopped working because class name of the affected element has changed to something that looks dynamically generated  (`.DYjDB-T3iPGc-ihIZgd`), so I changed it to target its child element